lib/unifex/code_generator/base_types/string.ex

```diff
@@ -11,7 +11,7 @@ defmodule Unifex.CodeGenerator.BaseTypes.String do
 
   @impl BaseType
   def generate_native_type(ctx) do
-    optional_const = if ctx.mode == :const, do: "const ", else: ""
+    optional_const = if ctx.mode in [:const, :const_if_not_ptr_on_ptr], do: "const ", else: ""
```

Comment:
I think someone is going to forget about `:const_if_not_ptr_on_ptr` sooner or later, we need to handle that better

Reply:
The only other solution I can see, is to check, if native type is pointer on pointer, in every place, we would call `&generate_native_type/1` with `%{mode: :const_if_not_ptr_on_ptr} = ctx`, but you suggested in https://github.com/membraneframework/unifex/pull/73#discussion_r615811178 to do not do this